extra-libs/liburing/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME="liburing"
+PKGDES="Linux-native io_uring I/O access library"
```

Comment:
A Kernel async I/O access library for Linux

(Stop copying from Arch Linux...)